tests/rules/method.js

```diff
@@ -103,7 +103,14 @@ eslintTester.run("method", rule, {
         },
         {
             code: "(Async.checkAppReady = function() { return true; })();"
-        }        
+        },
+        {
+            code: "let endTime = (mapEnd || (e => e.delta))(this._data[this._data.length - 1]);",
+            parserOptions: { ecmaVersion: 6 }
+        },
+        {
```

Comment:
Can you raise a bug for this if we want to scan this? Happy to let both of these pass through for now as they were doing so... but I think we should consider in a bug etc.